examples/mpi/overview.ipynb

```diff
@@ -128,6 +129,7 @@
    "outputs": [],
    "source": [
     "%%px\n",
+    "#NBVAL_IGNORE_OUTPUT\n",
```

Comment:
why do you need this one here? did you get another error in this cell?

Reply:
No, I did not, but the same happened before. cell 3 failed ->added nbval there -> cell 4 failed -> I will remove it probably...Just want to see how this thing works.. I will not be surprised if this error is raised again in other cells... Give it some time to be tested and I will fix/remove redundant things...

Reply:
I think cells 3 and 4 should be the only ones needing the NBVAL_IGNORE_OUTPUT tag because that's where we've seen the error appear. Maybe due to changing `configuration` ? for reasons I can't really explain. It's however important not to add the NBVAL tag randomly in the other cells too . If the error starts appearing in other cells, I'm afraid we'll have to investigate deeper what's going on. Hopefully this won't be the case

Reply:
Yes totally agree, I will investigate next hours...

Reply:
Tests pass right now in pipelines.

Reply:
but nonetheless, you should drop this